ios/Integreat/Info.plist

```diff
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2020.3.1</string>
+	<string>0.1</string>
```

Comment:
So what's this version code?

Reply:
This is the version code which is used for builds from developers.